modules/label/label.go

```diff
@@ -21,6 +21,19 @@ type Label struct {
 	ExclusiveOrder int    `yaml:"exclusive_order,omitempty"`
 }
 
+type ErrInvalidLabelColor struct {
```

Comment:
It should be able to unwrap to something like `util.ErrInvalidArgument`

Reply:
Actually I think we should rewrite the "Label" modal to use fetch/JSON, then we do not need to introduce the new error type, and it will be more friendly to end users.

Will try